src/raygun.tracekit.jquery.js

```diff
@@ -55,7 +55,7 @@
 
     var keys = ['complete', 'error', 'success'], key;
     while(key = keys.pop()) {
-      if ($.isFunction(options[key])) {
+      if (typeof options[key] === "function") {
```

Comment:
This is the only change to the provider. The rest is related to Continuous Integration tests.